### PR TITLE
dockerfile for building verify trusty demo

### PR DESCRIPTION
**Pull request by danblitzhou**
based on seahorn-minsize-rel dockerfile;
pulls content from trusty and https://github.com/danblitzhou/verifyTrusty, then builds jobs source files for verifying.

### CODE REVIEW

File: docker/seahorn-verify-trusty.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    echo "PS1='\${debian_chroot:+(\$debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\n\[\033[00m\]\\\$ '" >> /home/usea/.bashrc
+
+WORKDIR /opt
+
+RUN echo "pulling verify trusty environment" && \
+    # installing repo
+    mkdir ~/bin && PATH=~/bin:$PATH && \
+    curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo && \
+    chmod a+x ~/bin/repo && \
+    git clone https://github.com/danblitzhou/verifyTrusty.git && \
```
Comment (agurfinkel) [inline docker/seahorn-verify-trusty.Dockerfile]: can this use our `verifyTrusty.git` instead of your local one?

**Comment by agurfinkel** [comment]
@danblitzhou This is great, but it does not belong to SeaHorn repo. Can we put it into `verifyTrusty` repo instead?

File: docker/seahorn-verify-trusty.Dockerfile (hunk 1 above)
Comment (danblitzhou) [reply docker/seahorn-verify-trusty.Dockerfile]: I have pushed the code to there https://github.com/agurfinkel/verifyTrusty

**Comment by agurfinkel** [comment]
Merged into verifyTrusty repo

**Pull request closed without merging**